ProteoformSuiteInternal/ProteoformRelation.cs

```diff
@@ -51,22 +51,83 @@ public List<ProteoformRelation> mass_difference_group
                 this.group_count = value.Count;
                 this.group_adjusted_deltaM = value.Select(r => r.delta_mass).Average();
                 _mass_difference_group = value;
+                this.accepted = set_accepted(); //dependent on peak count
             }
         }
-        public int unadjusted_group_count { get; set; }
+
+        public int unadjusted_group_count { get; set; } //this is never set to anything? 
+        public string accession { get; set; } //theoretical for ET, null for EE
+        public string name { get; set; } //theoretical for ET, null for EE
+        public string fragment { get; set; }
+        public string ptm_list { get; set; }
+        public double proteoform_mass_1 { get; set; } //experiment for ET, experiment_1 for EE
+        public double proteoform_mass_2 { get; set; } //theoretical for EE, experiment_2 for EE
+        public double agg_intensity_1 { get; set; } //experiment for ET, experiment_1 for EE
+        public double agg_intensity_2 { get; set; } //0 for ET, experiment_2 for EE
+        public double agg_RT_1 { get; set; } //experiment for ET, experiment_1 for EE
+        public double agg_RT_2 { get; set; } //0 for ET, experiment_2 for EE
+        public int num_observations_1 { get; set; }
+        public int num_observations_2 { get; set; } //0 for ET
+        public int lysine_count { get; set; }
         public double group_adjusted_deltaM { get; set; }
+        public int running_sum { get; set; }
```

Comment:
"Running sum" is redundant of the "unadjusted group count." I think running_sum is quite vague. It took me a while to figure out what it meant when working through this code.